timon/configure.py

```diff
@@ -13,6 +13,7 @@
 from __future__ import absolute_import, print_function
 
 import os
+import sys
```

Comment:
os, sys, json, logging should be sorted alphabetically.
between line 17 and 18 there should be an empty line